client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -74,16 +74,14 @@ class ActivityLogItem extends Component {
 				<ActivityActor
 					{ ...pick( log, [ 'actorAvatarUrl', 'actorName', 'actorRole', 'actorType' ] ) }
 				/>
-				{ ! activityDescription && (
-					<div className="activity-log-item__title">{ activityTitle }</div>
-				) }
 				{ activityDescription && (
 					<div className="activity-log-item__description">
 						{ activityDescription.map( ( part, key ) => (
 							<FormattedBlock key={ key } content={ part } />
 						) ) }
 					</div>
 				) }
+				{ activityTitle && <div className="activity-log-item__title">{ activityTitle }</div> }
```

Comment:
A bit surprising to see this move down and apparently render alongside `activityDescription`.

Do we expect _either_ `getDescription` or `getTitle` to produce a value from an item, but not both?

Reply:
this was a response to external changes. we expect to have both a description and title now, and the title goes below